scene/main/window.cpp

```diff
@@ -807,6 +821,9 @@ void Window::set_visible(bool p_visible) {
 	updating_child_controls = false;
 
 	Viewport *embedder_vp = get_embedder();
+	if (embedder_vp && !(embedder_vp->is_embedding_subwindows() || !native)) {
```

Comment:
```suggestion
	if (embedder_vp && !embedder_vp->is_embedding_subwindows() && native) {
```
Simpler and equivalent